SPRINTLOG.md

```diff
@@ -344,6 +344,10 @@ _Nothing merged in CLI during this sprint_
 - Minor update jinja2 package to address cve ([#1503](https://github.com/ScilifelabDataCentre/dds_web/pull/1503))
 - Minor update jwcrypto package to address cve ([#1504](https://github.com/ScilifelabDataCentre/dds_web/pull/1504))
 
+# 2023-01-15 - 2024-01-25
```

Comment:
Isn't this a wrong sprint. It should go in the last one, i.e. the current spring